### PR TITLE
Add guests picker modal

### LINKED ISSUE #426: Refactor guests popup
[Zeplin](https://app.zeplin.io/project/5a1308cdae8ae89cae5c15af/screen/5a43f7b3d4e8333c75a655df)

### PR DESCRIPTION
**Pull request by tbergq**
closes #426

### CODE REVIEW

File: app/hotels/src/allHotels/searchForm/guests/Guests.js [hunk 1]
```diff
@@ -48,47 +43,25 @@ export const ButtonTitle = ({
...
-          onChange={this.handleChange}
-          onClose={this.handlePopupToggle}
-          isVisible={isPopupOpen}
-        />
-      </View>
-    );
-  };
+export default function Guests({ guests, openGuestsModal }: Props) {
+  return (
+    <View>
```
Comment (mrtnzlml) [inline app/hotels/src/allHotels/searchForm/guests/Guests.js]: Is this view necessary? I tried to remove it and it looks fine.

File: app/hotels/src/allHotels/searchForm/guests/GuestsModalScreen.js [hunk 2]
```diff
@@ -0,0 +1,229 @@
+// @flow
```
Comment (mrtnzlml) [inline app/hotels/src/allHotels/searchForm/guests/GuestsModalScreen.js]: Is it a good idea to hide these screen components somewhere in the package? I created `navigation` folder for this (but not sure what is better).

File: app/hotels/src/allHotels/searchForm/guests/GuestsModalScreen.js [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+          childCount={this.state.guests.children.length}
+          handleAdultChange={this.handleAdultChange}
+          handleChildrenChange={this.handleChildrenChange}
+        />
+      </View>
+      {this.state.guests.children.length > 0 && (
+        <View style={styles.container}>
+          <View style={styles.childAgeTitle}>
+            <Text style={styles.childAgeTitleText}>
+              <Translation id="HotelsSearch.GuestsModal.ChildAgeTitle" />
```
Comment (mrtnzlml) [inline app/hotels/src/allHotels/searchForm/guests/GuestsModalScreen.js]: Space between this title and `+-` button is too big. Check the design, please.

File: app/hotels/src/allHotels/searchForm/guests/GuestsModalScreen.js (hunk 2 above)
Comment (tbergq) [reply app/hotels/src/allHotels/searchForm/guests/GuestsModalScreen.js]: I think I will move these modal screens also to the navigation folder. Makes sense.

**Comment by tbergq** [comment]
@mrtnzlml Fixed, I also added scroll to end function to show the error message, since it could be outside of the screen depending on number of children and screen size.

**Pull request merged**